featuretools/tests/synthesis/test_deep_feature_synthesis.py

```diff
@@ -1368,3 +1370,48 @@ def test_primitive_options_instantiated_primitive(es):
             assert all(entity == 'stores' for entity in entities)
         elif isinstance(f.primitive, Mean):
             assert 'stores' not in entities
+
+
+def test_primitive_options_commutative(es):
+    class AddThree(TransformPrimitive):
+        name = 'add_three'
+        input_types = [Numeric, Numeric, Numeric]
+        return_type = Numeric
+        dask_compatible = True
+
+        def __init__(self, commutative=True):
+            self.commutative = commutative
```

Comment:
it doesn't look like we are setting `commutative` to `False` anywhere, perhaps we can skip making an `__init__` method and have it be a class attribute